### DISCUSSION TITLE
Github pages does not display repositories other than the original

### DISCUSSION THREAD

**Discussion by lufers** [community/community#23092] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I am new to git, github and github pages.</p>
<p>I did set up GitHub pages and got my main link http://&lt;your-github-sername&gt;.github.io.</p>
<p>It is my understanding that if I add a repository name at the end of the link, the content of the repository should be displayed on the browser.</p>
<p>http://&lt;your-github-username&gt;.github.io/repository-name</p>
<p>However this does not happen, as whenever I put  a repository name the browser send the error message 404 as below:</p>
<p>404</p>
<p><strong>File not found</strong></p>
<p>The site configured at this address does not contain the requested file.</p>
<p>If this is your site, make sure that the filename case matches the URL.<br>
For root URLs (like <a href="http://example.com/" rel="nofollow noopener">http://example.com/</a>) you must provide an index.html file.</p>
<p>I put my new_repository into the folder where my <a href="http://username.github.io" rel="nofollow noopener">username.github.io</a> is located, I did commit and put the changes of my new repository and still I am not able to make it display my website. </p>
<p>Could someone help me please.</p>
<p>Thanks</p>

**Comment by Kukun7610** [comment]
<p>Because u didn’t have specific Directory on your Host repo</p>
<p>for example,  </p>
<ul>
<li>
<p>you need to make a repo like <a href="http://USERNAME.github.io" rel="nofollow noopener">USERNAME.github.io</a> and its URL should be like ( <a href="https://github.com/USERNAME/USERNAME.github.io" rel="nofollow noopener">https://github.com/USERNAME/USERNAME.github.io</a> )</p>
</li>
<li>
<p>under the “<a href="http://USERNAME.github.io" rel="nofollow noopener">USERNAME.github.io</a>” u need to make subfolder to access it, if u try to access <a href="https://github.com/USERNAME/XYZ-project" rel="nofollow noopener">https://github.com/USERNAME/XYZ-project</a> then u could not access because the “XYZ-project” is not in your host repository u need to make sub-folder for that on your host for it 1st like (<a href="https://github.com/USERNAME/USERNAME.github.io/XYZ-project" rel="nofollow noopener">https://github.com/USERNAME/USERNAME.github.io/XYZ-project</a>)</p>
</li>
<li>
<p>make sure everything is on the same branch “master”</p>
</li>
</ul>
<p> that’s it, now good to go</p>
<p>for more information plz visit <a href="https://pages.github.com/" title="GitHub Pages" rel="nofollow noopener">https://pages.github.com/</a></p>
<p>Hope these help! Can’t wait to see host page :smiley:</p>
<p>Happy Coding!</p>

**Comment by winstliu** [comment, marked as answer]
<p>To have your repository show up at <a href="https://username.github.io/your-repository" rel="nofollow noopener">https://username.github.io/your-repository</a>, you need to setup GitHub Pages for that repository as well.</p>

**Reply by Ekanem11** [reply]
how do i do that

**Reply by Luv2Proxy** [reply]
Very late, but go to project settings and on the sidebar click pages

**Comment by francois1791** [comment]
<p>Make sure your html file is called “index.html”</p>

**Comment by matttali09** [comment]
<p>so i have a problem that the index page is showing up but not the other two pages. </p>
<p>my index.html shows up fine and when i build it with my code editor the other two pages work, but not on the <a href="http://github.io" rel="nofollow noopener">github.io</a> site that was created.</p>

**Comment by LandonJames** [comment]
<p>Sorry just to add to your issue as well but when my page is deployed it only shows the html portion and not the css portion <a href="https://landonjames.github.io/excursion/" rel="nofollow noopener">https://landonjames.github.io/excursion/</a></p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/landonjames">@landonjames</a>,</p>
<p>Since this is a different problem, you’d probably get better help by starting a new topic. However, there are a few topics with similar issues. <a href="https://github.community/t/issue-connecting-my-index-html-with-css/10417">Maybe this one might help</a>?</p>
<p>Thanks!</p>

**Comment by Mariusz071** [comment]
<p><a class="mention" href="/matttali09">@matttali09</a></p>
<p>I came across same issue, have you found a solution?</p>
<p>I got all my sub-pages working except particular one.</p>
<p>Obviously, locally everything works fine.</p>
<p>edit:</p>
<p>Sorted, in my case I was using by mistake “a” tag and href attribute instead of using “to” attribute and Link component.</p>

**Comment by matttali09** [comment]
<p><a class="mention" href="/mariusz071">@mariusz071</a> I had a similar solution, I didn’t realize the accuracy required for github vs open from files. I had my other pages capitalized in the anchor tag when the files were not capitalized in their actual name</p>

**Comment by ThirdEyeRose** [comment]
<p>You’ll find issues like that in more places than just GitHub because of how different operating systems handle things like capitals and special characters. Before working at GitHub, I was a web developer for many years and you wouldn’t believe how easy it is to have one of those mistakes lead to a “It works on my machine - why doesn’t it work here?” problem.</p>

**Comment by ammubharatram** [comment]
<p>This solved for me, thanks!</p>

**Comment by RubesWorld** [comment]
<p><strong>I WAS PULLING MY HAIR OUT</strong></p>
<p>I was following the tutorial from <a>M1mo</a> and <strong>could not figure out</strong> why their easy, 5 step tutorial to get the site up and running was going so smooth while I kept on getting that darn 404 message. My page was set up properly (<a href="http://user.github.io" rel="nofollow noopener">user.github.io</a>), gt pages were set up and the files were there and it was working on my local. Then, I dug a little bit deeper.</p>
<p><strong>IT’S ALL IN THE DETAILS</strong></p>
<p>Turns out my a href was pointing to <strong>I</strong> ndex with a <em>CAPITAL</em> ‘i’ while everything in my html files was to a lowercase ‘i’. It wasn’t working because I was pointing to the wrong thing from the very first a href in my header.</p>
<p><strong>TL:DR</strong></p>
<p>If your Index.html has a capital and you are pointing to index.html without a capital, it won’t work. Pay attention, code newbies.</p>

**Comment by theNoobDev10** [comment]
<p>Hey i did created new repo and also created a new folder locally inside my main repo folder and my index file name is index.html. But i still i’m getting 404 page not found. Am i missing something? </p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/imsree">@imsree</a>,</p>
<p>Is your repository public? If so, post a link and we can try to troubleshoot.</p>
<p>Thanks! </p>

**Comment by manhhieu1990** [comment]
<p>Awesome, I had similar problem and figured out its cause is exactly like you say. Just change Index.html to index.html and it works like a charm. Thank you.</p>

**Comment by KruttikaS** [comment]
<p>I’m getting similar problem like above. I crossed checked my files again, but the file name and anchor tag name is similar like in small alphabets but still its not working.Please tell me the solution.</p>

**Comment by Pagante** [comment]
<p>Hello, Top of the day to you! How are you and everyone at your end?</p>
<p>I hope you are all fine, and you are not affected with this pandemic. II am having some issues publishing my site on Github. Can i send the link down so you help me?</p>

**Comment by pavankmr1** [comment]
<p><a href="https://github.com/pavankmr1/firstrepo.git" rel="nofollow noopener">https://github.com/pavankmr1/firstrepo.git</a></p>

**Comment by the22ndmeusoc** [comment]
<p>Still not working. I have it setup i.e. <a href="https://github.com/the22ndmeusoc/the22ndmeusoc.github.io/tree/master/squads/xml" rel="nofollow noopener">https://github.com/the22ndmeusoc/the22ndmeusoc.github.io/tree/master/squads/xml</a> there is no index.html or anything like that. There is only squad.xml. That’s by design. The site is not published, only for the default. Also, AFAIK, there is a requirement for “insecure” http, so to speak, so when I configure as such, I do not see the site being published. Do I need to register a domain name or something like that?</p>

**Comment by oziomatanisha** [comment]
<p>Good day…I have same issue as well.<a href="https://t.co/t2Edi25UDX" rel="nofollow noopener">https://t.co/t2Edi25UDX</a>. My pages aren’t displaying</p>

**Comment by majidakhoulani** [comment]
i have same as this problem how to solve it please ??

**Comment by msbatin** [comment]
The General screen can be found under: BROWSE > Repositories > <project-name> > Genera
